### ISSUE TITLE
Can jcodec combine multiple mp4 files and generate one combined file?

### ISSUE THREAD

**Issue by HadiDortaj** [jcodec/jcodec#481]
I have multiple mp4 files and all of theme contain both video and audio tracks. I want to combine all of theme and create a single mp4 file containing all of those. Is it possible using jcodec? if so, could you please give me a sample code to do this? Thanks

**Comment by svitvitskiy** [comment]
It depends on a player. Technically you can combine multiple tracks into one, but realistically I doubt many players will support that. However, if those individual fragments came out of the same video, say they were split by DASH tool, you can safely combine them back into one file. Let if the latter is your case, I might be able to come up with a code sample.

**Comment by HadiDortaj** [comment]
Yes. They all point to the same file. Take the Instagram app, for example. When users share a video longer than one minute, Instagram splits that video into several videos up to a minute long. I want to convert these files into a single file.


> It depends on a player. Technically you can combine multiple tracks into one, but realistically I doubt many players will support that. However, if those individual fragments came out of the same video, say they were split by DASH tool, you can safely combine them back into one file. Let if the latter is your case, I might be able to come up with a code sample.

**Comment by HadiDortaj** [comment]
@svitvitskiy So could you please give me a sample code or a reference to do that? Thanks in advance.

**Comment by madhan-dev-k** [comment]
Hi @svitvitskiy, have you written any sample code for this? and please tell me can we combine different resolution videos using jcodec
